app/src/main/res/values/strings.xml

```diff
@@ -165,6 +165,8 @@
     <string name="browser_menu_stop">Stop</string>
     <!-- Browser menu button that opens the addon manager -->
     <string name="browser_menu_add_ons">Add-ons</string>
+    <!-- Browser menu button that opens account settings or -->
+    <string name="browser_menu_account_settings">Account info</string>
```

Comment:
You might want to consider creating a PR to preland the string since string freeze is tomorrow if you're expecting to  ship this fix for the upcoming release.

Reply:
+1 to this suggestion. Looking over this in-depth now, but I expect we will have at least one review cycle so it would be a safe bet to preland the string.

Reply:
Good suggestion! I wish I didn't miss it yesterday.